float64[7] left_arm_cart
float64[7] right_arm_cart
float64[2] speed
---
bool success